### DISCUSSION TITLE
Certain videos after download are Solid Colors

### DISCUSSION THREAD

**Discussion by MaestroMcCauley** [aclap-dev/video-downloadhelper#484] [Help and Q&A]
### Your question

I'm downloading a video from OnlyFans and after seeing it size 1.8 GB and opening it, it's nothing but alternating solid colors.  
Do I need to update something?  Other videos from other sites on the same platform are successful.  Is there a new security in place that blocks capture?

### Addon & CoApp version and details

⊕
Video DownloadHelper 8.2.2.8 (stable)
⊖
Platform
Platform Mac aarch64
Browser Mozilla Firefox 125.0.3
⊖
Companion App installed
Found companion app: VdhCoApp 2.0.19
Companion app binary: /Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp

**Comment by jcfrog** [comment]
hi,
I heard some videos on this site are protected by DRMs.
